woocommerce/woocommerce-android
# Issue/m2 wp media library

Adds the WP media library as a source for product images.

### Testing

- Open product detail and tap to add an image
- Tap "Add photos"
- Choose "WordPress media library" as the source
- Choose a few images and verify the activity title updates to show the selection count
- Press the back button and verify the discard dialog appears
- Long press an image and verify it shows fullscreen
- Rotate the device and ensure the selection is maintained
- Tap "Done" and verify the selected images appear on the product images screen
- Return to product detail, tap "Done" and verify the selected images are actually attached to the product

### Demos

Light mode and dark mode are very similar.

![light](https://user-images.githubusercontent.com/3903757/80624769-26070b00-8a1a-11ea-8a06-da29cf0e22ea.png)

![ezgif-4-b9dfbe17b75f](https://user-images.githubusercontent.com/3903757/80624784-2acbbf00-8a1a-11ea-950d-7ad1ab4ada12.gif)

Update release notes:
